### PR TITLE
Configuration of multiqc report

### PR DESCRIPTION
**Pull request by sofstam**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/multiqc_config.yml [hunk 1]
```diff
@@ -55,3 +89,171 @@ top_modules:
...
+    avg_sequence_length: "Length Input Reads"
+    percent_gc: "% GC Input Reads"
+    percent_duplicates: "% Dups Input Reads"
+    percent_fails: "% Failed Input Reads"
+  FastQC (post-Trimming):
+    total_sequences: "Nr. Processed Reads"
+    avg_sequence_length: "Length Processed Reads"
+    percent_gc: "% GC Processed Reads"
+    percent_duplicates: "% Dups Processed Reads"
+    percent_fails: "%Failed Processed Reads"
```
Comment (jfy133) [inline assets/multiqc_config.yml:243]:
```suggestion
    percent_fails: "% Failed Processed Reads"
```

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -55,3 +89,171 @@ top_modules:
...
+  Falco (post-Trimming):
+    total_sequences: "Nr. Processed Reads"
+    avg_sequence_length: "Length Processed Reads"
+    percent_gc: "% GC Processed Reads"
+    percent_duplicates: "% Dups Processed Reads"
+    percent_fails: "%Failed Processed Reads"
```
Comment (jfy133) [inline assets/multiqc_config.yml:249]:
```suggestion
    percent_fails: "% Failed Processed Reads"
```

**Review by jfy133 — APPROVED** [review]
LGTM, we can refine this over time :+1:

**Comment by sofstam** [comment]
I can merge once all tests pass.

**Pull request merged**